advertise.md

```diff
@@ -0,0 +1,86 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+<div class="advertise">
+  <div class="advertise container">
+    <h2>Technologies</h2>
+    <div class="border row">
+      <div class="border col-md-4 col-sm-4" style="height:200px;">
+        <div class="panel-content">
+          <label class="_default-text" style="margin-top: 20px;">
+            <img src="./assets/themes/zeppelin/img/spark_logo.jpg" width="140px">
+          </label>
+          <label class="_hover-text">
+            Spark interpreter group which consists of five interpreters which are Spark, Pyspark, SparkR, Spark SQL, Dependency loader.<br/>
+            <a href="./docs/0.8.0-SNAPSHOT/quickstart/spark_with_zeppelin.html" class="panel-button">USE NOW <span class="glyphicon glyphicon-chevron-right"></span></a>
+          </label>                     
+        </div>
+      </div>
+      <div class="border col-md-4 col-sm-4" style="height:200px;">
+        <div class="panel-content">
+          <label class="_default-text" style="font-size:56px;">SQL</label>
```

Comment:
4. not sure how we can improve this `SQL` `panel-content`.  But here are few suggestions. 

- Use a different `font-family` and `font-weight`
- `Roboto` with `font-weight: 100` is the best for me.
- you can test more fonts in https://fonts.google.com/?category=Sans+Serif,Monospace&thickness=2

### `SQL` text with `font-weight: 100`

![image](https://cloud.githubusercontent.com/assets/4968473/26776777/9924f428-4a15-11e7-86d8-87ed90970d22.png)

### `SQL` text with `font-weight: 500` (normal)

![image](https://cloud.githubusercontent.com/assets/4968473/26776740/67033d4c-4a15-11e7-9676-3f7305682394.png)